nearmint/tests/test.rs

```diff
@@ -0,0 +1,193 @@
+use node_runtime::state_viewer::AccountViewCallResult;
+use protobuf::Message;
+use serde_derive::{Deserialize, Serialize};
+use std::process::{Child, Command};
+use std::thread;
+use std::time::Duration;
+
+/// Test node that contains the subprocesses of tendermint and nearmint.
+/// Used for shutting down the processes gracefully.
+struct TestNode<'a> {
```

Comment:
Can we put it into the testlib?

Reply:
This is an ad hoc structure for testing nearmint specifically. If we want to integrate with the test infrastructure it will take more work and should ideally be done in a separate PR.